TGC/events/JANFlavor.txt

```diff
@@ -892,20 +892,24 @@ country_event = {
 		any_country = { 
 			limit = { 
 				government = theocracy
-				NOT = { tag = MON }
-				NOT = { tag = ITA }
-				NOT = { tag = PAP }
-				NOT = { tag = TIB }
-				NOT = { tag = DAG }
-				NOT = { tag = TOO }
-				NOT = { tag = JAL }
-				NOT = { tag = YUG }
-				NOT = { tag = TOU }
-				NOT = { tag = SUD }
-				NOT = { tag = TPG }
-				NOT = { tag = MGL }
-				NOT = { tag = ATH }
-				NOT = { tag = NYE }
+				NOT = { tag = AFG } #Afghanistan
```

Comment:
Unnecesary checks, use instead

`NOT = {
tag = AFG
tag = ATH
tag = CYR
.....`

And so on.

Reply:
OR

You could simply use

`NOT = { government = theocracy exists = yes }`